native/src/routeCalculationProgress.cpp

```diff
@@ -97,7 +98,34 @@ void RouteCalculationProgress::updateStatus(float distanceFromBegin, int directS
 	this->reverseSegmentQueueSize = reverseSegmentQueueSize;
 }
 
+float RouteCalculationProgress::getLinearProgressHH() {
+	float progress = 0;
+
+	for (int i = HHIteration::HH_NOT_STARTED; i <= HHIteration::DONE; i++) {
```

Comment:
According to best practices on C++ better to add to HHIteration "Count" item

Reply:
Count added